curation_concerns-models/app/models/concerns/curation_concerns/generic_file/derivatives.rb

```diff
@@ -0,0 +1,33 @@
+module CurationConcerns
+  module GenericFile
+    module Derivatives
+      extend ActiveSupport::Concern
+
+      included do
+        include Hydra::Derivatives
+        # This was taken directly from Sufia's GenericFile::Derivative.
+        # Using File and ServiceFile so this class will not generate two derivatives 
+        # with the same RDF URI. Is there a mime type ontology we should be using instead?
+        directly_contains_one :ogg, through: :files, type: ::RDF::URI("http://pcdm.org/use#ServiceFile"), class_name: "Hydra::PCDM::File"
```

Comment:
If this approach seems on the right path maybe we should make this part of the standard install generator as a flag someone can select to scaffold out a/v derivatives if they want.